### PR TITLE
I added Henry the fish to the the list of fish

### PR DESCRIPTION
**Pull request by stevebrownlee**
Martin wanted his fish Henry added to the web site.  I added the properties and the image that Martin provided.

## Steps to Test

1. Pull this branch
2. Open in Chrome
3. Verify that Henry is the second fish in the list
4. Make sure the image URL works and displays

### CODE REVIEW

**Review by heymonicakay — APPROVED** [review]
Pulled down branch srb-make-henry. Inspected code in VS Code and in browser, all expected changes were present. Thanks Steve!

**Review by heymonicakay — CHANGES_REQUESTED** [review]
Hey Steve! Thanks for making these changes, please update the spelling of 'Henry' to 'Henri' per client's request.

File: index.html [hunk 1]
```diff
@@ -29,14 +29,14 @@ <h2 class="header header--fishes">Fishes</h2>
                     <div class="fish__food">Tacos</div>
                 </div>
                 <div class="fish">
-                    <div class="fish__name">Geraldice</div>
+                    <div class="fish__name">Henry</div>
```
Comment (ajawashington) [inline index.html]: Henry needs to be Henri

**Review by ajawashington — CHANGES_REQUESTED** [review]
Added comment to this review! View all the changes!

**Pull request merged**